deeplenstronomy/check.py

```diff
@@ -400,8 +401,13 @@ def check_input_distributions(self):
                                     errs.append("DISTRIBUTIONS." + userdist + ".STEP must be a positive integer")
                         # if params are specified, they must be in a list
                         if 'PARAMS' in self.config["DISTRIBUTIONS"][userdist].keys():
-                            if not isinstance(self.config["DISTRIBUTIONS"][userdist]['PARAMS'], list):
+                            if not isinstance(self.config["DISTRIBUTIONS"][userdist]['PARAMS']:
```

Comment:
This should be `if not isinstance(self.config["DISTRIBUTIONS"][userdist]['PARAMS'], list):`. At present, it is a syntax error and will not run. Make sure to test that the code works before committing.

Reply:
Resolved by e310684